routetable/route_table.go

```diff
@@ -390,14 +473,14 @@ func (r *RouteTable) syncRoutesForLink(ifaceName string) error {
 			// In case this IP is being re-used, wait for any previous conntrack entry
 			// to be cleaned up.  (No-op if there are no pending deletes.)
 			r.waitForPendingConntrackDeletion(cidr.Addr())
-			if err := r.dataplane.RouteAdd(&route); err != nil {
+			if err := nl.RouteAdd(&route); err != nil {
 				logCxt.WithError(err).Warn("Failed to add route")
 				updatesFailed = true
 			}
 		}
 		if r.ipVersion == 4 && target.DestMAC != nil {
 			// TODO(smc) clean up/sync old ARP entries
-			err := r.dataplane.AddStaticArpEntry(cidr, target.DestMAC, ifaceName)
+			err := r.addStaticArpEntry(cidr, target.DestMAC, ifaceName)
```

Comment:
If this member function was still capitalized, it would be a bit harder for someone to accidentally (and wrongly) change this to ```err := addStaticArpEntry(cidr, target.DestMAC, ifaceName)```  Just a thought.

(And I believe that that would still compile, because the non-test impl `addStaticArpEntry` is also in the routetable package.)

Reply:
I'm not keen on making the field public again.  I tried a few spelling variants but didn't find any I liked.  If you do sub in the wrong one then the UT should fail with a permissions error so I think we should live with.